### PR TITLE
[Fixes Issue #9959] C programming resource in bangla added

### PR DESCRIPTION
**Pull request by khalid586**
## What does this PR do?
Add resource(s)

### Why is this valuable (or not)?
Good quality.

### How do we know it's really free?
Available on youtube. 

### For book lists, is it a book? For course lists, is it a course? etc.
This is a course available on youtube.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by khalid586** [comment]
@eshellman have a look at it and merge it if everything's okay. thanks

File: courses/free-courses-bn.md [hunk 1]
```diff
@@ -54,6 +54,7 @@
 * [C - All you need to know](https://www.youtube.com/playlist?list=PL_XxuZqN0xVASsjyqiNzgjUWHbDkN2Scy) - Stack Learner
 * [C Programming Bangla Tutorial Course](https://www.youtube.com/playlist?list=PLgH5QX0i9K3pCMBZcul1fta6UivHDbXvz) - Anisul Islam
 * [C Programming Bangla Tutorial](https://youtube.com/playlist?list=PLdl6zXgLsy3zwNjSMiYlOZOr20sykTfgo) - Bangla Coding Tutor
+* [C Programming Bangla Tutorial For Beginners 2023](https://youtube.com/playlist?list=PLNMnAEqLBwmrwDSycdTLsvZBhmK5kOtgV&si=N4c5Snw-UjlxG_k9) - Hablu Programmer
```
Comment (eshellman) [inline courses/free-courses-bn.md:57]:
```suggestion
* [C Programming Bangla Tutorial For Beginners 2023](https://youtube.com/playlist?list=PLNMnAEqLBwmrwDSycdTLsvZBhmK5kOtgV) - Hablu Programmer
```
tracking parameter

**Comment by khalid586** [comment]
@eshellman made the changes.

**Comment by khalid586** [comment]
@eshellman please merge the PR if there is no issue with the PR.

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**